threestudio/systems/imagedreamfusion.py

```diff
@@ -363,3 +384,6 @@ def on_test_epoch_end(self):
             name="test",
             step=self.true_global_step,
         )
+        shutil.rmtree(
+            os.path.join(self.get_save_dir(), f"it{self.true_global_step}-test")
+        )
```

Comment:
Are you sure this doesn't mess up the wandb upload? It happens asynchronously IIRC.
IMHO it would be best to add an option to delete most of the output folder for the run, after Trainer() is completely done training.

Reply:
it doesn't affect it. We do this for `on_validation_epoch_end` as well. Without it, in addition to test.mp4, there also remains an extra directory with all the images used to make test.mp4